radio/src/audio_arm.cpp

```diff
@@ -511,9 +515,16 @@ void audioTask(void * pdata)
 }
 #endif
 
-void mixSample(audio_data_t * result, int sample, unsigned int fade)
+void mixSample(audio_data_t * result, int sample, unsigned int fade, bool swVolumeAdjust = true)
 {
-  *result = limit(AUDIO_DATA_MIN, *result + ((sample >> fade) >> (16-AUDIO_BITS_PER_SAMPLE)), AUDIO_DATA_MAX);
+  *result = limit(AUDIO_DATA_MIN, *result + ((sample >> fade) >> (16 - AUDIO_BITS_PER_SAMPLE)), AUDIO_DATA_MAX);
+
+#if defined(SOFTWARE_VOLUME)
```

Comment:
Why do you do this here? I would see the SOFTWARE_VOLUME after all mixing is done